clickhouse/clickhouse-config.xml

```diff
@@ -3,6 +3,9 @@
         <level>warning</level>
         <console>true</console>
     </logger>
+    
+    <!-- Don't listen on IPv6 since it's not avaiable in Docker -->
+    <listen_host>0.0.0.0</listen_host>
```

Comment:
It is available if people decide to enable it :)

Reply:
I meant that we shouldn't disable it by default. We can document it though and keep this line commented out.

Reply:
Added some inline documentation. Feel free to modify as needed.

Reply:
Thank you!